src/pages/claim-details/claim-actions.tsx

```diff
@@ -261,22 +281,33 @@ export default function ClaimActions(props: Props) {
                 {' counter of '}
                 <br />${formatUsd(claim.settlementAmountInUsd!)}
               </div>
-              <div className={styles.actionPanel}>
-                <Button variant="secondary" disabled={disableActions} onClick={() => setModalToShow('appeal')}>
-                  Appeal
-                </Button>
-                <Modal open={modalToShow === 'appeal'} onClose={handleModalClose}>
-                  <AppealConfirmation
-                    disputeId={claim.dispute!.id}
-                    disableActions={disableActions}
-                    onConfirm={handleAppeal}
-                    onCancel={handleModalClose}
-                  />
-                </Modal>
-              </div>
-              <p className={styles.actionMessage}>
-                You can appeal within the given time frame or the counter offer will be automatically accepted.
-              </p>
+              {dispute.period === 'Appeal' && (
+                <>
+                  <div className={styles.actionPanel}>
+                    <Button variant="secondary" disabled={disableActions} onClick={() => setModalToShow('appeal')}>
+                      Appeal
+                    </Button>
+                    <Modal open={modalToShow === 'appeal'} onClose={handleModalClose}>
+                      <AppealConfirmation
+                        disputeId={claim.dispute!.id}
+                        disableActions={disableActions}
+                        onConfirm={handleAppeal}
+                        onCancel={handleModalClose}
+                      />
+                    </Modal>
+                  </div>
+                  <p className={styles.actionMessage}>
+                    You can appeal within the given time frame or the counter offer will be automatically accepted.
+                  </p>
+                </>
+              )}
+              {dispute.period === 'Execution' && (
+                <div className={styles.actionPanel}>
+                  <Button variant="secondary" disabled={disableActions} onClick={handleExecutePayout}>
+                    Execute Payout
```

Comment:
Shouldn't the user know that there will be a bot executing these rulings and this is only useful if the bot is down (or they want to execute the ruling immediately)?

Reply:
> or they want to execute the ruling immediately?

Correct, the user should be able to execute the ruling themselves if the bot is delayed, so I don't think we need to warn them about the bot.